src/main/java/net/dv8tion/jda/api/managers/ChannelManager.java

```diff
@@ -235,10 +235,114 @@ default ChannelManager putPermissionOverride(@Nonnull IPermissionHolder permHold
+
+    /**
+     * Adds an override for the specified role with the provided permission sets as allowed and denied permissions. If
+     * the role already had an override on this channel it will be replaced instead.
+     *
+     * @param  roleId
+     *         The ID of the role to set permissions for
+     * @param  allow
+     *         The permissions to grant, or null
+     * @param  deny
+     *         The permissions to deny, or null
+     *
+     * @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+     *         If the currently logged in account does not have {@link net.dv8tion.jda.api.Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS}
+     *         in this channel, or tries to set permissions it does not have without having {@link Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS} explicitly for this channel through an override.
+     *
+     * @return ChannelManager for chaining convenience
+     *
+     * @see    #putRolePermissionOverride(long, long, long)
+     * @see    java.util.EnumSet EnumSet
+     */
+    @Nonnull
+    @CheckReturnValue
+    default ChannelManager putRolePermissionOverride(long roleId, @Nullable Collection<Permission> allow, @Nullable Collection<Permission> deny) {
+        long allowRaw = allow == null ? 0 : Permission.getRaw(allow);
+        long denyRaw  = deny  == null ? 0 : Permission.getRaw(deny);
+        return putRolePermissionOverride(roleId, allowRaw, denyRaw);
+    }
+
+    /**
+     * Adds an override for the specified member with the provided raw bitmasks as allowed and denied permissions. If the
+     * member already had an override on this channel it will be replaced instead.
+     *
+     * @param  memberId
+     *         The ID of the member to set permissions for
+     * @param  allow
+     *         The bitmask to grant
+     * @param  deny
+     *         The bitmask to deny
+     *
+     * @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+     *         If the currently logged in account does not have {@link net.dv8tion.jda.api.Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS}
+     *         in this channel, or tries to set permissions it does not have without having {@link Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS} explicitly for this channel through an override.
+     *
+     * @return ChannelManager for chaining convenience
+     *
+     * @see    #putMemberPermissionOverride(long, Collection, Collection)
+     * @see    net.dv8tion.jda.api.Permission#getRaw(Permission...) Permission.getRaw(Permission...)
+     */
+    @Nonnull
+    @CheckReturnValue
+    ChannelManager putMemberPermissionOverride(long memberId, long allow, long deny);
+
+    /**
+     * Adds an override for the specified member with the provided permission sets as allowed and denied permissions. If
+     * the member already had an override on this channel it will be replaced instead.
```

Comment:
```suggestion
     * Adds an override for the specified member with the provided permission sets as allowed and denied permissions.
     * If the member already had an override on this channel it will be replaced instead.
```